src/locate.rs

```diff
@@ -2,8 +2,17 @@ use std::path::PathBuf;
 
 use crate::Result;
 
-pub fn locate_steam_dir() -> Result<PathBuf> {
-    locate_steam_dir_helper()
+pub fn locate_steam_dir() -> Result<Vec<PathBuf>> {
+    #[cfg(target_os = "linux")]
+    return locate_steam_dir_helper();
```

Comment:
```suggestion
    locate_steam_dir_helper()
```

I think this can have the explicit return dropped